src/monitors.ts

```diff
@@ -0,0 +1,180 @@
+    }
+    if (!monitorParams.name) {
+      monitorParams.name = SERVERLESS_MONITORS[serverlessMonitorId].name;
+    }
+  }
+  return monitorParams;
+}
+
+/**
+ * Checks to see if the given monitor is a serverless recommended monitor
+ * @param serverlessMonitorId - Unique ID string defined for each serverless monitor
+ * @returns true if a given monitor is a serverless recommended monitor
+ */
+function checkIfServerlessMonitor(serverlessMonitorId: string) {
+  return Object.keys(SERVERLESS_MONITORS).includes(serverlessMonitorId);
+}
+
+/**
+ * Checks to see if the monitor already exists
+ * @param serverlessMonitorId - Unique ID string defined for each serverless monitor
+ * @param existingMonitors - Monitors that have already been created
+ * @returns true if given monitor already exists
+ */
+function doesMonitorExist(serverlessMonitorId: string, existingMonitors: { [key: string]: number }) {
+  return Object.keys(existingMonitors).includes(serverlessMonitorId);
+}
+
+/**
+ * Deletes the monitors that have been removed from the plugin
+ * @param pluginMonitors Monitors that are currently defined in the plugin
+ * @param existingMonitors Monitors that have already been created
+ * @param monitorsApiKey API Key
+ * @param monitorsAppKey Application Key
+ * @returns an array of successfully deleted monitors
+ */
+async function deleteRemovedMonitors(
+  pluginMonitors: Monitor[],
+  existingMonitors: { [key: string]: number },
+  monitorsApiKey: string,
+  monitorsAppKey: string,
+) {
+  const successfullyDeletedMonitors: string[] = [];
+  const currentMonitorIds: string[] = [];
+  pluginMonitors.forEach((currentMonitor) => currentMonitorIds.push(Object.keys(currentMonitor)[0]));
+  for (const pluginMonitorId of Object.keys(existingMonitors)) {
+    if (!currentMonitorIds.includes(pluginMonitorId)) {
+      const response = await deleteMonitor(
+        existingMonitors[pluginMonitorId],
+        pluginMonitorId,
+        monitorsApiKey,
+        monitorsAppKey,
+      );
+      if (response) {
+        successfullyDeletedMonitors.push(` ${pluginMonitorId}`);
+      }
+    }
+  }
+  return successfullyDeletedMonitors;
+}
+
+/**
+ * Creates, updates, and deletes the appropriate monitor configurations as defined in the serverless.yml file
+ * @param monitors - Monitors defined in the serverless.yml file
+ * @param monitorsApiKey - the API Key
+ * @param monitorsAppKey - the Application Key
+ * @param cloudFormationStackId - the Cloud Formation Stack ID
+ * @param service - the Service
+ * @param env - the Environment
+ * @returns monitors that have been successfully created, updated, and deleted according to the configuration defined in the plugin
+ */
+export async function setMonitors(
+  monitors: Monitor[],
+  monitorsApiKey: string,
+  monitorsAppKey: string,
+  cloudFormationStackId: string,
+  service: string,
+  env: string,
+) {
+  const serverlessMonitorIdToMonitorId = await getExistingMonitors(
```

Comment:
I'm a little bit confused about what this constant means - `serverlessMonitorIdToMonitorId` to me sounds like it would be a function that converts one type of ID to another, but that doesn't seem to be what this is.